py3status/py3.py

```diff
@@ -502,25 +502,32 @@ def prevent_refresh(self):
         """
         self._module.prevent_refresh = True
 
-    def notify_user(self, msg, level='info', rate_limit=5):
+    def notify_user(self, msg, level='info', rate_limit=5, title='py3status', icon=None):
```

Comment:
(Note for fam) I wonder if it is good to change default behavior from `title=py3status` to...
`title=None` + (`(if None` + `title=py3status: {module_name}`) to see module name in the notification too.... or maybe better with just `title={module_name}` (not related to py3status core).